opengrok-indexer/src/main/java/org/opengrok/indexer/web/messages/Message.java

```diff
@@ -50,7 +50,44 @@
     @NotEmpty(message = "tags cannot be empty")
     private Set<String> tags = Collections.singleton(MESSAGES_MAIN_PAGE_TAG);
 
-    private String cssClass = "info";
+    public enum CssClassType {
+        /**
+         * Known values: SUCCESS, INFO, WARNING, ERROR.
+         * The values are sorted according to their level. Higher numeric value of the level (i.e. the enum ordinal)
+         * means higher priority.
+         */
+        SUCCESS("success"), INFO("info"), WARNING("warning"), ERROR("error");
+
+        private String cssClassString;
+
+        CssClassType(String str) {
+            cssClassString = str;
+        }
+
+        public static CssClassType StringToCssClassType(String val) throws IllegalArgumentException {
+            if (val == null) {
+                return INFO;
```

Comment:
this should be illegal argument as well no? the caller should check for null and assing default INFO

Reply:
This was probably implementation ~convenience~laziness.